ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -3095,6 +3095,27 @@ keys:
               type: bool
               description: Global proxy settings should be enabled before enabling
                 per-vlan
+  ip_name_servers:
+    documentation_options:
+      filename: data_model/Management
+    type: list
+    primary_key: ip_address
```

Comment:
I don't think we want `primary_key` here. Some customers use both inband and oob at the same time, so they need to reach the same IP via different VRFs

Reply:
Modified and added tests in [eb30aa6](https://github.com/aristanetworks/ansible-avd/pull/2738/commits/eb30aa645230115c564aee4cc3aebd48d878d3dc)